processor/src/test/java/org/mapstruct/ap/test/fluent/getters/Issue1601Test.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.test.fluent.getters;
+
+import org.mapstruct.ap.spi.AccessorNamingStrategy;
+import org.mapstruct.ap.spi.BuilderProvider;
+import org.mapstruct.ap.spi.ImmutablesAccessorNamingStrategy;
+import org.mapstruct.ap.test.fluent.getters.domain.ImmutableItem;
+import org.mapstruct.ap.test.fluent.getters.domain.Item;
+import org.mapstruct.ap.test.fluent.getters.dto.ImmutableItemDto;
+import org.mapstruct.ap.test.fluent.getters.dto.ItemDto;
+import org.mapstruct.ap.testutil.IssueKey;
+import org.mapstruct.ap.testutil.ProcessorTest;
+import org.mapstruct.ap.testutil.WithClasses;
+import org.mapstruct.ap.testutil.WithServiceImplementation;
+import org.mapstruct.ap.testutil.WithServiceImplementations;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@WithClasses({
+    ItemMapper.class,
+    Item.class,
+    ItemDto.class,
+    ImmutableItem.class,
+    ImmutableItemDto.class
+})
+@IssueKey("1601")
+@WithServiceImplementations({
+    @WithServiceImplementation(provides = BuilderProvider.class, value = Issue1601BuilderProvider.class),
+    @WithServiceImplementation(provides = AccessorNamingStrategy.class, value = ImmutablesAccessorNamingStrategy.class)
+})
+public class Issue1601Test {
+
+    @ProcessorTest
+    public void shouldIncludeBuildType() {
+        var item = ImmutableItemDto.builder().id( "test" ).build();
+
+        var target = ItemMapper.INSTANCE.map( item );
```

Comment:
Please do not use `var`. We don't have a check for this, but I am not a fan of it and we are not using it anywhere else in our codebase.